pom.xml

```diff
@@ -0,0 +1,678 @@
+                                <text>https://github.com/teragrep/teragrep/blob/main/LICENSE</text>
+                            </all>
+                            <family>Teragrep</family>
+                        </license>
+                    </licenses>
+                    <consoleOutput>true</consoleOutput>
+                    <excludeSubProjects>false</excludeSubProjects>
+                    <excludes>
+                        <!-- git -->
+                        <exclude>.git/**</exclude>
+                        <exclude>.gitattributes</exclude>
+                        <exclude>.gitignore</exclude>
+                        <exclude>.gitmodules</exclude>
+                        <!-- ci -->
+                        <exclude>.github/workflows/*</exclude>
+                        <exclude>.github/ISSUE_TEMPLATE/*</exclude>
+                        <exclude>.github/PULL_REQUEST_TEMPLATE/*</exclude>
+                        <exclude>toolchains.xml</exclude>
+                        <exclude>settings.xml</exclude>
+                        <!-- maven -->
+                        <exclude>pom.xml</exclude>
+                        <exclude>eclipse-java-formatter.xml</exclude>
+                        <!-- readme -->
+                        <exclude>README.adoc</exclude>
+                    </excludes>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                        <phase>test</phase>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+    <profiles>
+        <profile>
+            <id>publish-maven-central</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-gpg-plugin</artifactId>
+                        <version>1.6</version>
+                        <executions>
+                            <execution>
+                                <id>sign-artifacts</id>
+                                <phase>verify</phase>
+                                <goals>
+                                    <goal>sign</goal>
+                                </goals>
+                                <configuration>
+                                    <gpgArguments>
+                                        <arg>--pinentry-mode</arg>
+                                        <arg>loopback</arg>
+                                    </gpgArguments>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>flatten-maven-plugin</artifactId>
```

Comment:
flatten-maven-plugin might as well be inside main build and not profile

Reply:
Solved in commit 79e7942bab31d31302d436da42c0e065ce22da57